PurchasesTests/SubscriberAttributes/PurchasesSubscriberAttributesTests.swift

```diff
@@ -140,15 +140,15 @@ class PurchasesSubscriberAttributesTests: XCTestCase {
 
         setupPurchases()
 
-        expect(self.mockOperationDispatcher.invokedDispatchOnWorkerThreadCount) == 1
         expect(self.mockBackend.invokedGetSubscriberDataCount) == 1
-        expect(self.mockSubscriberAttributesManager.invokedSyncAttributesForAllUsersCount) == 0
+        expect(self.mockDeviceCache.cachePurchaserInfoCount) == 1
```

Comment:
Isn't this line the same as the next one?

Reply:
they're actually different, although the naming could be improved - one is for the `read` method and the other one for the `write`